services/navigation/builtin/builtin.go

```diff
@@ -539,6 +596,7 @@ func (svc *builtIn) startWaypointMode(ctx context.Context, extra map[string]inte
 
 			wp, err := svc.store.NextWaypoint(ctx)
 			if err != nil {
+				time.Sleep(time.Millisecond * 50)
```

Comment:
This sleep is to limit busy waiting, not required for correctness but just prevents wasting CPU

Reply:
They both happen to be 50 ms but are not directly coupled. I've made another constant for this sleep frequency.

Reply:
In either case its the frequency at which we check for new plans correct?  In one case its the one where there is an error getting next waypoint (in practice because there are none) and in the other its that we're polling the history of a known waypoint?  I'd be ok collapsing them but this is fine too

Reply:
I've made them one frequency